Author: Nicholas Bamber <nicholas@periapt.co.uk>
Subject: problems with mod_perl
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=618469
Last-Update: 2011-03-16
--- a/httpd/conf/httpd.conf
+++ b/httpd/conf/httpd.conf
@@ -6,27 +6,17 @@
 # The w3c-markup-validator postinst script will link
 # /etc/apache2/conf.d/w3c-markup-validator.conf -> /etc/w3c/httpd.conf
 #
-# To run the validator without mod_perl 2.x even if it is loaded, comment
-# out the "IfDefine MODPERL2" sections below.
+# Note: mod_perl has been disabled. See #618469.
+#
 
 # First, tell httpd that check and sendfeedback.pl are CGI scripts.
 
 AliasMatch ^/+w3c-validator/+check(/+referer)?$   /usr/lib/cgi-bin/w3c-markup-validator/check
 AliasMatch ^/+w3c-validator/+feedback(\.html)?$ /usr/lib/cgi-bin/w3c-markup-validator/sendfeedback.pl
 
-<IfDefine MODPERL2>
-# Note: this affects the whole server, not just the validator.
-PerlSwitches -T
-</IfDefine>
-
 <LocationMatch "^/+w3c-validator/+(check(/+referer)?|feedback(\.html)?)$">
   Options               +ExecCGI
   SetHandler            cgi-script
-<IfDefine MODPERL2>
-  SetHandler            perl-script
-  PerlResponseHandler   ModPerl::Registry
-  PerlOptions           +ParseHeaders
-</IfDefine>
   # Environment settings affecting validator's outside connectivity, see
   # LWP::UserAgent and Net::FTP documentation for details.
   #SetEnv                http_proxy      http://proxy:3128
